src/main/resources/application.properties

```diff
@@ -31,6 +31,9 @@ cdc.pay-wallets-log-events.project=\${CDC_LOG_EVENTS_PROJECT}
 # Retry policy for sending message
 cdc.retry-send.maxAttempts=\${CDC_SEND_RETRY_MAX_ATTEMPTS}
 cdc.retry-send.intervalInMs=\${CDC_SEND_RETRY_INTERVAL_IN_MS}
+# Retry policy for stream connection
+cdc.retry-stream.maxAttempts=\${CDC_STREAM_RETRY_MAX_ATTEMPTS}
+cdc.retry-stream.intervalInMs=\${CDC_STREAM_RETRY_INTERVAL_IN_MS}
```

Comment:
add this properties also to this [property file](https://github.com/pagopa/pagopa-payment-wallet-cdc-service/blob/main/docker-local/wallet-cdc/application.properties) (it's a duplicated files that is used only in integration tests

Reply:
Done in [993499b](https://github.com/pagopa/pagopa-payment-wallet-cdc-service/pull/16/commits/993499bcbb6eec10e5d065b47cc504407dd7dc8e)